sources/web/datalab/polymer/modules/file-manager/file-manager.ts

```diff
@@ -42,9 +42,14 @@ interface Notebook {
 }
 
 enum DatalabFileType {
-  DIRECTORY,
-  FILE,
-  NOTEBOOK,
+  DIRECTORY = 'directory',
+  FILE = 'file',
+  NOTEBOOK = 'notebook',
+}
+
+enum DatalabFileStatus {
+  IDLE = '',
+  RUNNING = 'Running',
```

Comment:
Fixed by using a method for now, we'll think about moving that to a strings module.